python3: can't open file 'stdout.py': [Errno 2] No such file or directory
